### PR TITLE
chore: add notebooks for POST /collections

### PR DESCRIPTION
**Pull request by danieljhegeman**
Functional: @Bento007 
For awareness: @brianraymor 

Add: 
- Python notebook with sample code for `POST /collections`
- R notebook with sample code for `POST /collections`

### CODE REVIEW

File: notebooks/curation_api/create_collection_dev_R.ipynb [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bea213f4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "domain_name <- \"cellxgene.dev.single-cell.czi.technology\"\n",
```
Comment (Bento007) [inline notebooks/curation_api/create_collection_dev_R.ipynb:83]: This should point to prod by default. We don't want to advertise the dev environment to the curators.

**Review by Bento007 — CHANGES_REQUESTED** [review]
Since these are user-facing notebooks, do we want to have dev in the file names?

File: notebooks/curation_api/create_collection_dev_R.ipynb (hunk 1 above)
Comment (Bento007) [reply notebooks/curation_api/create_collection_dev_R.ipynb:83]: This also applies to the python note book.

Comment (Bento007) [reply notebooks/curation_api/create_collection_dev_R.ipynb:83]: or we need to tell the users where to find the domain name they need. It's not obvious that they can use the same domain from the frontend.

Comment (danieljhegeman) [reply notebooks/curation_api/create_collection_dev_R.ipynb:83]: agreed, will change

Comment (danieljhegeman) [reply notebooks/curation_api/create_collection_dev_R.ipynb]: done

**Review by Bento007 — APPROVED** [review]

**Pull request merged**